### ISSUE TITLE
DATABASE SELECTOR at top of wateva window - GO-AROUND

### ISSUE THREAD

**Issue by DingBEN** [pawelsalawa/sqlitestudio#3654]
### Details
**Sweet and short**
The way around that _pesky useless database selector that one must always chase_
is to make sure all queries state the database:  **FROM database1.table**
that way if the selector is on database2 ... it saves me the constant **CLONK**
_...as mentioned in other posts, **TRASH this approach**, it is cumbersome and useless, only gets in the way
[ unless you implement it appwide... **I now have trashed it, will never use it and will never worry about it, no more CLONKs** ]_

**BASICALLY all your windows are tied to a query and its database**
This amounts to changing your approach and make the app QUERYcentric(data) and let a list of queries be picked from a system table to move around.   ...that also takes care of queries that address multiple databases.  **...a 'current' database has not much meaning.**
